### PR TITLE
Amin/end to end/event hours by admin on adding event

### PR DESCRIPTION
**Pull request by AminesCodes**
event hours by admin on adding event
## Description

event hours by admin on adding event
## Motivation and Context

A better way to track volunteers hours
## How Has This Been Tested?

End to end
## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: backend/routes/events.js [hunk 1]
```diff
@@ -159,9 +175,15 @@ router.post('/add', async (req, res, next) => {
                 location: processInput(req.body.location, 'hardVC', 'location', 200),
                 instructor: processInput(req.body.instructor, 'hardVC', 'instructor', 100),
                 numberOfVolunteers: processInput(req.body.numberOfVolunteers, 'idNum', 'number of volunteers'),
-                materialsUrl: processInput(req.body.materialsUrl, 'softVC', 'materials url')
+                materialsUrl: processInput(req.body.materialsUrl, 'softVC', 'materials url'),
+                eventDuration: processInput(req.body.eventDuration, 'idNum', 'event duration')
             }
     
+            const calcEventTime = calcHours(eventData.start, eventData.end);
+            if (calcEventTime < calcEventTime.eventDuration) {
```
Comment (joseph-p-pasaoa) [inline backend/routes/events.js:183]: this should be:
if (calcEventTime < **eventData**.eventDuration) {

File: backend/routes/events.js [hunk 2]
```diff
@@ -191,9 +213,15 @@ router.put('/edit/:event_id', async (req, res, next) => {
...
                 instructor: processInput(req.body.instructor, 'hardVC', 'instructor', 100),
                 numberOfVolunteers: processInput(req.body.numberOfVolunteers, 'idNum', 'number of volunteers'),
-                materialsUrl: processInput(req.body.materialsUrl, 'softVC', 'materials url')
+                materialsUrl: processInput(req.body.materialsUrl, 'softVC', 'materials url'),
+                eventDuration: processInput(req.body.eventDuration, 'idNum', 'event duration')
             }
-    
+            
+            const calcEventTime = calcHours(eventData.start, eventData.end);
+            if (calcEventTime < calcEventTime.eventDuration) {
```
Comment (joseph-p-pasaoa) [inline backend/routes/events.js:221]: same as above here, misnamer

File: backend/routes/events.js (hunk 1 above)
Comment (AminesCodes) [reply backend/routes/events.js:183]: Good catch, thank you

File: backend/routes/events.js (hunk 2 above)
Comment (AminesCodes) [reply backend/routes/events.js:221]: haha, thanks :)

**Pull request merged**